src/core/icons/clock.tsx

```diff
@@ -0,0 +1,11 @@
+export const SvgClock = () => {
+	return (
+		<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+			<path
+				fillRule="evenodd"
+				clipRule="evenodd"
```

Comment:
`clip-rule` [does not apply in this case](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/clip-rule). There’s no `clip-path`.

```suggestion
```

Reply:
Thanks Max, I've removed 👍 (and squashed into the same commit)